impress-org/givewp
# Feature: makes bulk actions from donation list table extensible

## Description



This PR adds a way to extend the default core bulk actions. This is useful for addons that need to implement custom bulk actions on the donation list table like in this case: https://github.com/impress-org/give-pdf-receipts/pull/247 

## Affects



Bulk actions from the donations list table.

## Testing Instructions



After pulling [this PR](https://github.com/impress-org/give-pdf-receipts/pull/247) and active the PDF Receipts add-on, follow these steps:

1. Go to Give > Donations and select a few donations by clicking on the checkbox aside from the ID column;
2. Select the "Download Receipts" on the bulk actions selector and click on the apply button;
3. You should be redirected to Give > Tools > PDF Export where the download will be handled.

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204470020369035
